user-docs/latest/content-manager/introduction-to-content-manager.md

```diff
@@ -0,0 +1,87 @@
+# Introduction to the Content Manager
+
+The Content Manager is a core plugin of Strapi: its is a feature that is always activated by default and cannot be deleted. It is accessible both when the application is in environment development, and in production.
+
+The Content Manager is divided into 2 categories, both displayed in the main navigation: *Collection types* and *Single types*. Each category contains the available collection and single content-types, which were created beforehand using the Content-Types Builder. From these 2 categories, admin panel users can create, manage and distribute content.
```

Comment:
Tagging @Aurelsicoko and @maevalienard so we can clarify as I'm honestly not sure. I've always used left menu, but that doesn't seem like the proper term in user docs.

Guessing Nav, Navigation, or Main Nav/Navigation.

Reply:
After discussing with Aurélien & Maeva, it seems we're all fine with using "main navigation".
It will, by the way, allow us to reuse "navigation" for the "navigation bar" which contains, among other options, the link to the administrator profile and the logout button.